transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -189,8 +197,44 @@ private Metacard createMetacard(InputStream input, Metadata metadata, String uri
         return metacard;
     }
 
-    private String toWkt(String lon, String lat) {
+	private String getCoordinates(String cornerCoordinates, String corner) {
+		int startIndex = cornerCoordinates.indexOf(corner); 
+		StringBuilder coordinates = new StringBuilder();
+		boolean start = false;
+		boolean end = false;
+		if (startIndex != -1) {
+			startIndex += (corner).length();
+			for (int i = startIndex; i < cornerCoordinates.length(); i++) {
+				if (cornerCoordinates.charAt(i) == '(') {
+					start = true;
+				} else if (cornerCoordinates.charAt(i) == ')') {
+					break;
+				} else if (start) {
+					coordinates.append(cornerCoordinates.charAt(i));
+				}
+			}
+		}        
+		return coordinates.toString().trim();
+	}
+	
+	private String toWkt(String lowerLeftLat, String lowerLeftLon, String upperRightLat, String upperRightLon) {	
+        if (StringUtils.isBlank(lowerLeftLat) || StringUtils.isBlank(lowerLeftLon) ||
+		    StringUtils.isBlank(upperRightLat) || StringUtils.isBlank(upperRightLon))  {
+            return null;
+        }
 
+		StringBuilder wkt = new StringBuilder();
```

Comment:
It does not look like the StringBuilder is doing anything here.  Just do `String wkt = String.format(...)` instead.